solutions/instances/variables.tf

```diff
@@ -43,6 +43,151 @@ variable "prefix" {
   default     = null
 }
 
+##############################################################################
+# IBM Cloud Logs
+##############################################################################
+variable "cloud_logs_provision" {
+  description = "Provision a IBM Cloud Logs instance?"
+  type        = bool
+  default     = false
+}
+
+variable "cloud_logs_plan" {
+  type        = string
+  description = "The IBM Cloud Logs plan to provision. Available: standard"
+  default     = "standard"
+}
+
+variable "cloud_logs_instance_name" {
+  type        = string
+  description = "The name of the IBM Cloud Logs instance to create. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
+  default     = "cloud-logs"
+}
+
+variable "cloud_logs_region" {
+  description = "The IBM Cloud region where Cloud Logs instances will be created."
+  type        = string
+  default     = "eu-es"
+}
+
+variable "cloud_logs_tags" {
+  type        = list(string)
+  description = "Tags associated with the IBM Cloud Logs instance (Optional, array of strings)."
+  default     = []
+}
+
+variable "cloud_logs_service_endpoints" {
+  description = "The type of the service endpoint that will be set for the IBM Cloud Logs instance."
+  type        = string
+  default     = "public-and-private"
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.cloud_logs_service_endpoints)
+    error_message = "The specified service_endpoints is not a valid selection"
+  }
+}
+
+variable "enable_cloud_logs_data" {
+  type        = bool
+  description = "Enable data bucket for a IBM Cloud Logs instance"
+  default     = true
+}
+
+variable "enable_cloud_logs_metrics" {
+  type        = bool
+  description = "Enable metrics bucket for a IBM Cloud Logs instance"
+  default     = true
+}
+
+variable "enable_en_integration" {
```

Comment:
Name should show en integration with Cloud logs. How about using..`enable_en_cloud_logs_integration`